src/content/reference/docker-compose.mdx

```diff
@@ -208,6 +208,8 @@ either a string or a list. If it's a list, the first item must be either `NONE`,
 If it's a string, it's equivalent to specifying `CMD-SHELL` followed by that string.
 - `http` (object): Defines the path and port that has to be tested on the container to set
  the healthcheck as successful
+- `x-okteto-readiness` (bool): Defines if the probe should be a readiness probe (default: `true`). 
```

Comment:
what happens if both are true @jLopezbarb ?

Reply:
We will have a readiness and a liveness k8s healthcheck probe